docker-compose.yml

```diff
@@ -1,17 +1,36 @@
 version: "3.7"
 
 services:
+  certbot:
+    container_name: certbot
+    image: certbot/certbot:latest
+    depends_on:
+      - gateway
+    command: >-
+             certonly --reinstall --webroot --webroot-path=/var/www/certbot
+             --email ${EMAIL} --agree-tos --no-eff-email
+             -d ${DOMAIN}
+    volumes:
+      - ./gateway/etc/letsencrypt:/etc/letsencrypt
+      - ./certbot/data:/var/www/certbot
+
   gateway:
     image: sgs/gateway:${TAG:-v1.0.0}
     build:
       context: ./gateway/    
```

Comment:
we can have  separate `build.docker-compose.yml` to do the docker build in CI and publish the images to our docker hub workspace  and then have  `docker-compose.yml` to just  use published images

Reply:
Thanks @mozzy11 I will do that on the next iteration